pipeline.Dockerfile

```diff
@@ -36,11 +36,31 @@ RUN apt update -q \
     && apt autoremove -yqq --purge && apt clean && rm -rf /var/lib/apt/lists/* /var/log/*
 
 WORKDIR /kubespray
-COPY . .
 
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1 \
     && pip install --no-compile --no-cache-dir pip -U \
-    && pip install --no-compile --no-cache-dir -r tests/requirements.txt -r requirements.txt \
+    && pip install --no-compile --no-cache-dir \
```

Comment:
That's fine as well but fyi if using a "modern" version of docker/buildkit (not sure about kaniko) you can use the following syntax: `RUN --mount=type=bind,target=./requirements.txt,src=./requirements.txt [...]`